src/contracts/core/RewardsCoordinator.sol

```diff
@@ -267,8 +290,36 @@ contract RewardsCoordinator is
     }
 
     /// @inheritdoc IRewardsCoordinator
-    function setGlobalOperatorCommission(uint16 _globalCommissionBips) external onlyOwner {
-        _setGlobalOperatorCommission(_globalCommissionBips);
+    function setDefaultOperatorSplit(uint16 split) external onlyOwner {
+        _setDefaultOperatorSplit(split);
+    }
+
+    /// @inheritdoc IRewardsCoordinator
+    function setOperatorAVSSplit(
+        address operator,
+        address avs,
+        uint16 split
+    ) external onlyWhenNotPaused(PAUSED_OPERATOR_AVS_SPLIT) {
+        require(msg.sender == operator, "RewardsCoordinator.setOperatorAVSSplit: caller is not the operator");
+        require(split <= ONE_HUNDRED_IN_BIPS, "RewardsCoordinator.setOperatorAVSSplit: split must be <= 10000 bips");
+
+        uint32 activatedAt = uint32(block.timestamp) + activationDelay;
+        uint16 oldSplit = _getOperatorSplit(operatorAVSSplitBips[operator][avs]);
+        _setOperatorSplit(operatorAVSSplitBips[operator][avs], split, activatedAt);
+
+        emit OperatorAVSSplitBipsSet(msg.sender, operator, avs, activatedAt, oldSplit, split);
+    }
+
+    /// @inheritdoc IRewardsCoordinator
+    function setOperatorPISplit(address operator, uint16 split) external onlyWhenNotPaused(PAUSED_OPERATOR_PI_SPLIT) {
+        require(msg.sender == operator, "RewardsCoordinator.setOperatorPISplit: caller is not the operator");
+        require(split <= ONE_HUNDRED_IN_BIPS, "RewardsCoordinator.setOperatorPISplit: split must be <= 10000 bips");
+
+        uint32 activatedAt = uint32(block.timestamp) + activationDelay;
```

Comment:
move `activatedAt` logic into `_setOperatorSplit`?

Reply:
I did think about it when writing this. But it's being emitted in the event below as well. Then that would need a separate `uint32(block.timestamp) + activationDelay` calculation. 

I think this is cleaner.